packages/node/src/constants/api.ts

```diff
@@ -0,0 +1 @@
+export const DEFAULT_API_URL = "https://api.staging.tonightpass.com";
```

Comment:
vaut mieux pas en variables d'env ?

Reply:
Developers can choose their own environment variables using the baseURL parameter, which is better than imposing a standard.

Reply:
tu changeras pas de position tfaçon haha